scripts/read_ledger

```diff
@@ -109,6 +111,20 @@ def get_storage(type_, ledger_data_dir):
                                 read_only=True)
 
 
+def get_additional_storages(ledger_data_dir):
+    config = getConfig()
+    postfix = config.transactions_file_base_postfix
+
+    additional_storages = []
+    dir_names = [name for name in os.listdir(ledger_data_dir)
```

Comment:
Can't we just get the list of `additional_storages` by something like
```
additional_storages = [dir_name[:dir_name.find(postfix) for dir_name in os.listdir(ledger_data_dir) if postfix in dir_name]
```

Reply:
ok